lib/views/recent-commits-view.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {autobind} from 'core-decorators';
+
+class RecentCommitView extends React.Component {
+  static propTypes = {
+    commit: PropTypes.object.isRequired,
+  };
+
+  render() {
+    return (
+      <li className='github-RecentCommit'>
+        <img className='github-RecentCommit-avatar'
+          src="https://avatars3.githubusercontent.com/u/7910250?v=4&s=32"
```

Comment:
Default avatar (when there is no GitHub avatar) can be added with

```
src="atom://github/img/avatar.svg"
```

Should look like

![screen shot 2018-02-27 at 3 11 07 pm](https://user-images.githubusercontent.com/378023/36713418-d9a16ba2-1bd0-11e8-9f83-d612820ffa7a.png)